packages/bot-web-ui/src/stores/journal-store.js

```diff
@@ -41,12 +55,32 @@ export default class JournalStore {
             error_message = error && error.error ? error.error.message : message;
         }
 
-        this.messages.unshift({ date, time , message: error_message, message_type });
-        this.messages = this.messages.slice(0);  // force array update
+        this.unfiltered_messages.unshift({ date, time , message: error_message, message_type });
+        // this.filterMessage(this.checked_filters);
+    }
+
+    @computed
+    get filtered_message() {
```

Comment:
Could we make this plural for clarity?

```suggestion
    get filtered_messages() {
```